examples/traefik/docker-compose-swarm-traefik.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  traefik:
+    image: traefik:v2.11.8
+    ports:
+      - "80:80"
+    command:
+      - --api.insecure=false # set to 'false' on production
+      - --api.dashboard=true # see https://docs.traefik.io/v2.11/operations/dashboard/#secure-mode for how to secure the dashboard
+      - --api.debug=false # enable additional endpoints for debugging and profiling
+      - --log.level=INFO # debug while we get it working, for more levels/info see https://docs.traefik.io/observability/logs/
+      - --log.format=common
+      - --accesslog=true
+      - --accesslog.fields.names.StartUTC=drop
+      - --providers.docker=true
+      - --providers.docker.swarmMode=true
+      - --providers.docker.exposedbydefault=false
+      - --providers.docker.network=public
+      - --entryPoints.web.address=:80
+      - --entryPoints.web.transport.respondingTimeouts.readTimeout=0 # https://doc.traefik.io/traefik/v2.11/routing/entrypoints/#respondingtimeouts
+      - --entryPoints.web.transport.respondingTimeouts.idleTimeout=0
+      - --entryPoints.web.forwardedHeaders.insecure
+        # - --entryPoints.websecure.address=:443
+    environment:
+      - TZ=Asia/Shanghai
```

Comment:
SUGGESTION:

Again, make it neutral to the most amount of people.  `TZ=UTC` would be neutral.